internal/stacks/stack_manager.go

```diff
@@ -611,44 +610,44 @@ func (s *StackManager) createMember(id string, index int, options *types.InitOpt
 
 func (s *StackManager) StartStack(options *types.StartOptions) (messages []string, err error) {
 	fmt.Printf("starting FireFly stack '%s'... ", s.Stack.Name)
-	// Check to make sure all of our ports are available
+
+	// Check port availability
 	err = s.checkPortsAvailable()
 	if err != nil {
 		return messages, err
 	}
+
 	hasBeenRun, err := s.Stack.HasRunBefore()
 	if err != nil {
 		return messages, err
 	}
 	if !hasBeenRun {
-		setupMessages, err := s.runFirstTimeSetup(options)
-		messages = append(messages, setupMessages...)
+		// Pass FirstEvent to the first-time setup
+		setupMessages, err := s.runFirstTimeSetup(&types.InitOptions{
+			FromBlock: options.FromBlock,
+		})
```

Comment:
FromBlock doesn't exist in InitOptions, it should be StartOptions. Getting errors in the IDE